CONTRIBUTING.md

```diff
@@ -224,4 +224,3 @@ gotchas to bear in mind:
   `node_modules/balena-sdk/node_modules/balena-errors`  
   In the case of subclasses of `TypedError`, a string comparison may be used instead:  
   `error.name === 'BalenaApplicationNotFound'`
-
```

Comment:
This commit is practically empty (just this blank line deletion), and used to highlight the Release Notes URL on the commit message, at the top of the list of the v12.0.0 commits in the CLI changelog.